sdk/test-utils/testing-recorder-new/package.json

```
@@ -18,7 +18,7 @@
     "extract-api": "echo skipped",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "dev-tool run test:browser",
-    "integration-test:node": "dev-tool run test:node-ts-input --mocha=\"--timeout 1200000 'test/*.spec.ts'\"",
+    "integration-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 'test/*.spec.ts'",
```

Might be best to use escaped double quotes instead of single quotes now that they're not nested. My tests with cmd.exe made it seem like single quotes were handled... strangely... by cmd.

```suggestion
    "integration-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 \"test/*.spec.ts\"",
```